src/app/features/types.tsx

```diff
@@ -29,6 +29,7 @@ export type BagDataError = {
 }
 
 export type BrkData = {
+  bag_id: string | null
```

Comment:
`Components.Schemas.Address["bag_id"] | null`?

Reply:
Doesn't seem to exist. Couldnt find anything related in apiSchema.d.ts

Reply:
Ah, you are right. Thought this was zaken-frontend.